__tests__/app.test.js

```diff
@@ -66,6 +66,7 @@ describe("/api/reviews/:review_id", () => {
             category: "dexterity",
             created_at: expect.any(String), //can't check for the exact date
             votes: 5,
+            comment_count: 3,
```

Comment:
this should really be a new test rather than a refactor - your code is solid though :)